setup.py

```diff
@@ -10,11 +10,21 @@
 import shutil
 import sys
 
-from setuptools import setup, Extension  # pylint: disable=g-multiple-import
+from setuptools import setup
 
 # Path to directory containing setup.py
 here = os.path.abspath(os.path.dirname(__file__))
 
+# Get the pybind11 setup helpers
+#
+# This is appended, so if already available in site-packages, that is used
+# instead
+sys.path.append(os.path.join(here, "pybind11"))
+
+from pybind11.setup_helpers import Pybind11Extension  # pylint: disable=g-import-not-at-top
+
+del sys.path[-1]
```

Comment:
Can you add a comment explaining why this is necessary?

Reply:
It's just tidy. You could leave `$PWD/pybind11` in the path if you prefer.

Reply:
I'll remove it, since you don't clean up the build_scripts addition below either.